### DISCUSSION TITLE
Download music as files

### DISCUSSION THREAD

**Discussion by Adhf94** [SpotX-Official/SpotX#383] [Ideas]
Its it possible to download songs to reproduce without internet? and locally, like a flag file or something?

**Comment by amd64fox** [comment]
@Vexed092 
Do not recommend any software here that downloads audio streams directly from Spotify, as enough accounts get banned for it every day.

A [safe method](https://github.com/spotDL/spotify-downloader) for downloading music from Spotify.

**Comment by gmy77** [comment]
Let's be satisfied... we already have a wonderful thing... let's forget it or risk "ruining everything..."

**Reply by MikaTech-dev** [reply]
well said
